integration/build.gradle

```diff
@@ -46,6 +46,7 @@ dependencies {
     // Google dependencies
     constraints {
         implementation 'com.google.guava:guava:30.1.1-jre'
+        implementation 'org.json:json:20231013' // security vulnerability
```

Comment:
The integration directory likely is not scanned for vulnerabilities, but we might as well update its version of the library to match the service directory.